backend/src/enums/user.auth.provider.ts

```diff
@@ -1,5 +1,5 @@
 export enum UserAuthProvider {
-  INTERNAL = 'internal',
-  GOOGLE = 'google',
-  GITHUB = 'github',
+  INTERNAL = 1,
+  GOOGLE = 2,
+  GITHUB = 3,
```

Comment:
Whats the reason of this change?

Reply:
imo its better to store numbers than strings in db